src/plugins/discover/public/application/components/doc_views/surrounding_docs_app.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { i18n } from '@osd/i18n';
+import { useOpenSearchDashboards } from '../../../../../opensearch_dashboards_react/public';
+import { DiscoverServices } from '../../../build_services';
+import { getRootBreadcrumbs } from '../../helpers/breadcrumbs';
+import { SurroundingDocsView } from './surrounding_docs_view';
+import { IndexPattern } from '../../../opensearch_dashboards_services';
+
+export interface SurroundingDocsUrlParams {
+  id: string;
+  indexPatternId: string;
+}
+
+export function SurroundingDocsApp() {
+  const {
+    services: { chrome, indexPatterns },
+  } = useOpenSearchDashboards<DiscoverServices>();
```

Comment:
This should fix the broken breadcrumb issue. You will need to do the same for the single doc view

```suggestion
  const {
    services: {
      chrome,
      indexPatterns,
      core: {
        application: { getUrlForApp },
      },
    },
  } = useOpenSearchDashboards<DiscoverServices>();
  const baseUrl = getUrlForApp(PLUGIN_ID); 
```


And then down below where you set the breadcrumbs you can do:

```ts
...getRootBreadcrumbs(baseUrl),
```

Then in `src/plugins/discover/public/application/helpers/breadcrumbs.ts` update rootBreadcrumb as follows:

```ts
export function getRootBreadcrumbs(baseURL: string) {
  return [
    {
      text: i18n.translate('discover.rootBreadcrumb', {
        defaultMessage: 'Discover',
      }),
      href: baseURL,
    },
  ];
}
```